### PR TITLE
move test bastion services to ignition and systemd and mirror docker images

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

**Comment by enxebre** [comment]
retest this

File: contrib/govcloud/resources/gateway.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Nat Gateway routing
```
Comment (alexsomesan) [inline contrib/govcloud/resources/gateway.service]: Is this instance also supposed to act as a NAT gateway?

File: contrib/govcloud/main.tf [hunk 2]
```diff
@@ -57,16 +57,29 @@ resource "aws_instance" "bastion" {
   }
 }
 
-data "template_file" "services" {
-  template = "${file("${path.module}/services.sh")}"
+data "ignition_config" "main" {
+  files = ["${compact(list(
```
Comment (alexsomesan) [inline contrib/govcloud/main.tf]: I presume the compact(list(..)) come from a time when more than one entries were used here?

File: contrib/govcloud/README.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+This directory contains Terraform configuration that provisions a VPC with a VPN connection and a PowerDNS server in AWS GovCloud.
+This is setup is not production ready and is designed to emulate a customer-like deployment in order to end-to-end test deploying Tectonic in AWS GovCloud using a required pre-created internal VPC.
 
 This Terraform configuration provisions the following AWS resources by default:
 * 1 VPC with name configured by `TF_VAR_vpc_name`
 * 4 subnets in the VPC with count configured by `TF_VAR_subnet_count`
 * 1 public subnet containing an internet gateway and NAT gateway
 * 1 private Route 53 zone for `tectonic.dev.coreos.systems` configured by `TF_VAR_base_domain`
-* 1 t2.micro EC2 Container Linux instance in the public subnet running docker containers for OpenVPN, PowerDNS and Nginx for serving OpenVPN client configuration.
+* 1 t2.micro EC2 Container Linux instance in the public subnet running docker containers for OpenVPN, PowerDNS (and mysql) and Nginx for serving OpenVPN client configuration.
```
Comment (alexsomesan) [inline contrib/govcloud/README.md]: t2.micro might be kind of small for all those components.

File: contrib/govcloud/README.md [hunk 4]
```diff
@@ -1,13 +1,14 @@
 # Internal Cluster
 
-This directory contains Terraform configuration that provisions a VPC with a VPN connection and a PowerDNS server in AWS GovCloud. This setup is designed to emulate a customer-like deployment in order to end-to-end test deploying Tectonic as a private "Internal Cluster" to an "Existing VPC.
+This directory contains Terraform configuration that provisions a VPC with a VPN connection and a PowerDNS server in AWS GovCloud.
+This is setup is not production ready and is designed to emulate a customer-like deployment in order to end-to-end test deploying Tectonic in AWS GovCloud using a required pre-created internal VPC.
```
Comment (alexsomesan) [inline contrib/govcloud/README.md]: extra "is" - "This setup"

**Pull request merged**